src/main/java/flaxbeard/steamcraft/tile/TileEntityVacuum.java

```diff
@@ -31,15 +31,15 @@
 
 import java.util.List;
 
-public class TileEntityVacuum extends SteamTransporterTileEntity implements ISteamTransporter, IWrenchable, IWrenchDisplay {
-
+public class TileEntityVacuum extends SteamTransporterTileEntity
+        implements ISteamTransporter, IWrenchable, IWrenchDisplay {
```

Comment:
class definitions should be a single line, regardless of how many interfaces it implements.